docs/store/datastore.md

```diff
@@ -11,7 +11,7 @@ A data store defines a storage provider (e.g. file system, S3, Azure blob, Iguaz
 
 MLRun supports multiple data stores. (More can easily added by extending the `DataStore` class.)
 Data stores are referred to using the schema prefix (e.g. `s3://my-bucket/path`). The currently supported schemas and their urls:
-* **files** &mdash; local/shared file paths, format: `/file-dir/path/to/file`
+* **files** &mdash; local/shared file paths, format: `/file-dir/path/to/file` (Unix) or `C:/dir/file` (Windows)
```

Comment:
Don't include windows file path here

Reply:
@gilad-shaham why not? we discussed this with @theSaarco and agreed on this example following this issue https://github.com/mlrun/mlrun/issues/3028